modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItemsList.js

```diff
@@ -37,6 +39,19 @@ export default function CartItemsList() {
 
 			{cartState.accountId && !!id && <CartQuickAdd />}
 
+			{hasPriceOnApplication(cartItems) && (
```

Comment:
move this to  parameter of this component

Reply:
Done